api/cloudprotocol/cloudprotocol.go

```diff
@@ -192,33 +197,24 @@ type QueueInfo struct {
 
 // InstanceFilter instance filter structure.
 type InstanceFilter struct {
-	ServiceID string  `json:"serviceId"`
+	ServiceID string  `json:"serviceId,omitempty"`
```

Comment:
Then it should be pointer here. Otherwise, it is hard to distinguish if field is present or not.